src/vs/workbench/parts/files/electron-browser/fileActions.contribution.ts

```diff
@@ -187,7 +187,7 @@ MenuRegistry.appendMenuItem(MenuId.OpenEditorsContext, {
 	group: 'navigation',
 	order: 40,
 	command: copyPathCommand,
-	when: ResourceContextKey.HasResource
```

Comment:
@isidorn `IsFile` is a bit more specific and so it would only copy the path for files and remote resources.

Reply:
Yes, I also mentioned this in our chat. Makes perfect sens